docker-compose.yaml

```diff
@@ -73,19 +93,8 @@ services:
       - backend-prod
     networks:
       - circles-net
-  init-mongo:
-    # Utility container to populate / overwrite database
-    # Run after build or when you need to overwrite data
-    build:
-      context: ./backend
-      dockerfile: init-mongo.dockerfile
-    container_name: init-mongo
-    env_file:
-      - ./env/backend.env
-    depends_on:
-      - mongodb
-    networks:
-      - circles-net
+
+  # TODO-OLLI(pm): idek if this works or how it should work post init-mongo death
```

Comment:
I doubt it

Reply:
Oh i actually was thinking how to replace this, since i think ci mainly uses it to get the backend containers to run and cleaned...

I think if we can assume that the containers start empty, then we can just shift the ci to start backend directly (as itll run in dev, which at the moment only drops users). 
But, if we cannot, then i can make a simple new dockerfile that is just the same as the production backend docker (overwrites courses), but uses a new clarg to drop users too...

Reply:
I believe the purpose was something to do with handling the production instances of databases. Not 100% sure tho.

Reply:
interesting